packages/component/src/Styles/StyleSet/ScrollToEndButton.js

```diff
@@ -22,6 +22,7 @@ export default function createScrollToEndButtonStyle({
     outline: 0,
     padding: paddingRegular,
     position: 'absolute',
+    zIndex: 1, // We formed a stacking context in the parent container, so we can use "z-index" here.
```

Comment:
Unfortunately, we need to use `zIndex` to make the "New messages" button pop on top of all bubbles. This is because we re-ordered the DOM to put the "New messages" button between bubbles, instead of appending after all bubbles. We need the re-order to maintain tab order.

Reply:
Makes sense.